scheduler/src/cook/mesos.clj

```diff
@@ -207,6 +207,7 @@
                           (reify LeaderSelectorListener
                             (takeLeadership [_ client]
                               (log/warn "Taking mesos leadership")
+                              (swap! mesos-leadership-atom (constantly true))
```

Comment:
This was because `swap!` takes a function from old value -> new value, but for this we don't really care about the old value. Maybe there's a better way to express this?